payloadcms/payloadcms-dev/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  payload:
```

Can we name the image to know that it's the DEV one?

```suggestion
  payload-dev:
```